hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/FairCallQueue.java

```diff
@@ -58,7 +58,10 @@
 
   public static final Logger LOG = LoggerFactory.getLogger(FairCallQueue.class);
 
-  /* The queues */
+  /*
+  * Save the queue data of multiple priority strategies.
+  * Usually the number of queue data and priority strategies saved is the same.
+  *  */
   private final ArrayList<BlockingQueue<E>> queues;
```

Comment:
Also, good to convert this to `private final List<BlockingQueue<E>> queues` if you would like to consider as part of this PR.